apis/Google.Cloud.Logging.V2/Google.Cloud.Logging.Log4Net/GoogleStackdriverAppender.cs

```diff
@@ -289,6 +292,23 @@ private LogEntry BuildLogEntry(LoggingEvent loggingEvent)
             return logEntry;
         }
 
+        private void TryAddGitRevisionId(Dictionary<string, string> labels)
+        {
+            string gitId = null;
+            try
+            {
+                gitId = SourceContext.AppSourceContext?.Git?.RevisionId;
+            }
+            catch (Exception ex) when (ex is System.Security.SecurityException || ex is InvalidProtocolBufferException
+                || ex is InvalidJsonException || ex is UnauthorizedAccessException)
+            { }
+
+            if (!String.IsNullOrWhiteSpace(gitId))
+            {
+                labels.Add(SourceContext.GitRevisionIdLogLabel, gitId);
```

Comment:
I think I missed where `GitRevisionIdLogLabel` was added to `SourceContext`. Is that definitely the right place for it? It feels like a logging concern more than one of `SourceContext` itself...